### PR TITLE
fix: Guidance block content alignment and show arrow button

### PR DESCRIPTION
**Pull request by hqta**
# Objective
Product has request the following changes
- Show button arrow for small Guidance Block sizes. (RTL arrow orientation also addressed)
- Vertically center align the content when there is a tall Scene illustration

# Motivation and Context
https://cultureamp.atlassian.net/jira/software/projects/KDS/boards/289?selectedIssue=KDS-128

# Screenshots (if appropriate)
Show arrow for small Guidance Block
Screen Shot 2022-01-24 at 2 01 47 pm
Amend RTL arrow button orientation
Screen Shot 2022-01-24 at 2 04 15 pm
Vertically center content for tall Scene illustrations
Screen Shot 2022-01-24 at 3 44 01 pm

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

**Comment by lloydstubber** [comment]
@hqta this looks to be failing due to vulnerabilities. Also, can you pop a screenshot in the PR?

**Review by Tommy-Chen-cultureamp — APPROVED** [review]

File: draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx [hunk 1]
```diff
@@ -257,7 +261,7 @@ TwoStackedLayout.storyName = "Two Stacked Content Layout"
 export const StackedLayoutWithScene = () => (
   <GuidanceBlock
     layout="stacked"
-    illustration={<HumanityAtWork alt="" />}
+    illustration={<SkillsCoachManagerHub alt="" />}
```
Comment (mcwinter07) [inline draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx:264]: Thanks for adding the portrait illustration 👍 should be able to sort that out in the aspect ratio fix

**Review by mcwinter07 — APPROVED** [review]
The changes looks good to go 👍 Since there isn't a full sticker sheet in the designs for this yet we can probably hold off till its in the heart design kit. Merge away my friend!

**Comment by jmitsios** [comment]
@hqta Just took a look over it and looks great!  Stellar work on this. 👍

**Pull request merged**